itm_minimal_runner.py

```diff
@@ -172,7 +181,11 @@ def main():
                 kdma_training=args.kdma_training
             )
         while True:
```

Comment:
I personally am scared of using `while True:`. If the loop is broken `if scenario.session_complete` it may be better to use `while not scenario.session_complete` or alternatively a do while loop if the loop needs to be executed at least once